### ISSUE TITLE
"fill-opacity" style is ignored if "opacity" is less than 1

### ISSUE THREAD

**Issue by nuc134r** [ElinamLLC/SharpVectors#112]
Hello. Thank you for this beautiful library. 

We ran into this problem after update to version 1.3.0. Specification of `fill-opacity` style has no effect if `opacity` is less than 1. The minimal example I came up with is two rectangles taking two halves of the canvas. One with solid `fill-opacity` and another is half-transparent. Overall `opacity` for both is 0.99.

On version 1.2.0 it renders correctly:

![image](https://user-images.githubusercontent.com/13202642/66398192-065cb680-e9e6-11e9-9fb9-3cb112473ca8.png)

On version 1.3.0 it paints both rects with the same opacity:

![image](https://user-images.githubusercontent.com/13202642/66398205-0ceb2e00-e9e6-11e9-9343-d42498fc6c5e.png)

This is how Chrome renders this image, as a source of truth:

![image](https://user-images.githubusercontent.com/13202642/66398673-e1b50e80-e9e6-11e9-82e5-9895e4f2bafa.png)

The example SVG code:
```
<svg
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:cc="http://creativecommons.org/ns#"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:svg="http://www.w3.org/2000/svg"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   width="48px"
   height="48px"
   version="1.1">
  <g>
    <rect
       style="opacity:0.99;fill:#fe2b2b;fill-opacity:1;"
       width="24"
       height="48" />
  </g>
  <g>
    <rect
       style="opacity:0.99;fill:#fe2b2b;fill-opacity:0.5;"
       width="24"
       x="24"
       height="48" />
  </g>
</svg>
```

**Comment by paulushub** [comment]
@nuc134r Thanks for reporting this issue. I think this is now fixed in the latest commit. ff7f882
Please verify this.

**Comment by nuc134r** [comment]
Fixed in 1.4.0. Thank you!
